### DISCUSSION TITLE
Adições de raças para Caverna do Saber

### DISCUSSION THREAD

**Discussion by arturcorreaf** [YuriAlessandro/gerador-ficha-tormenta20#347] [Geral]
Percebi que há duas raças que não estão incluídas na Caverna do Saber, são elas:

Povos Trovão - DB #174
Moreau - DB #172

Também há uma reformulação da raça Golem na DB #182.

**Comment by YuriAlessandro** [comment]
Valeu, deixei escapar. Adicionado lá.
